config/default.yml

```diff
@@ -113,6 +113,14 @@ RSpec/ExampleLength:
   Enabled: true
   Max: 5
 
+RSpec/ImplicitExpect:
+  Description: Check that a consist implict expectation style is used.
```

Comment:
Thanks! For future reference you only need to flag something in the description on the class. The build enforces that this description is the same